### PR TITLE
Add doc links to console forms

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add documentation link to console forms.

This is a re-do from https://github.com/TheThingsNetwork/lorawan-stack/pull/3162 which messed up `pkg/webui/components/form/field/field.styl`.

Looks like:

Screenshot 2020-11-16 at 14 55 05

And here's the unbroken login and rights screens:

Screenshot 2020-11-16 at 14 58 57
Screenshot 2020-11-16 at 14 59 59

#### Testing

Run locally

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by kschiffer** [comment]
The help icon is displayed too high, it should be leveled with the field text, both on desktop and mobile. It did not look like that in the original branch.

**Comment by benolayinka** [comment]
> The help icon is displayed too high, it should be leveled with the field text, both on desktop and mobile. It did not look like that in the original branch.

Yep but that's because the original branch (´git checkout 20f81843d1e73ff089c01a5c297e78b3e78f321e`) centered the labels vertically, which also centered the icons, and broke the API keys screen and login. I think you'll be able to solve this much more easily than me and probably without breaking other stuff along the way, wanna try @kschiffer ?

Centered icons in the broken original branch:
Screenshot 2020-11-17 at 10 22 44

**Comment by asmulko** [comment]
@kschiffer, @benolayinka I've rebased v3.10 branch into this feature branch, had to resolve some conflicts. Also I've updated the positioning of the glossary-icon that @kschiffer mentioned in comments. If you could check if I have missed anything during conflict resolving or functionality wise - it would be great.

**Comment by benolayinka** [comment]
Looks like it works though I don't understand the CSS entirely - did you just make the line height such that the margin works?

Screenshot 2020-11-18 at 13 57 12

Rights screen is okay too

Screenshot 2020-11-18 at 13 55 54

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ For details about compatibility between different releases, see the **Commitment
 - Support for Application Activation Settings in the Join Server to configure Application Server KEK, ID and Home NetID.
 - Downlink queue invalidated message sent upstream by Application Server to support applications to re-encrypt the downlink queue when Application Server skips FRMPayload crypto.
 - Navigation to errored step in the end device wizard in the Console.
+- Rerence available glossary entries for form fields in the Console.
```
Comment (benolayinka) [inline CHANGELOG.md]:
```suggestion
- Reference available glossary entries for form fields in the Console.
```

File: pkg/webui/components/form/field/field.styl [hunk 2]
```diff
@@ -33,6 +33,9 @@
       width: 14rem
       font-weight: $fw.normal
 
+  .glossary-link
```
Comment (benolayinka) [inline pkg/webui/components/form/field/field.styl]: Is this necessary?

**Comment by asmulko** [comment]
> Looks like it works though I don't understand the CSS entirely - did you just make the line height such that the margin works?
> 
> Screenshot 2020-11-18 at 13 57 12
> 
> Rights screen is okay too
> 
> Screenshot 2020-11-18 at 13 55 54

I must admit CSS is not my strong side, so I assumed that line-height solution is good enough for the requirement.
Another way I can think of doing this would be using `display: flex` on the link and aligning items center.

If you prefer the second solution or there's a better way - please let me know. Thanks!

File: pkg/webui/components/form/field/field.styl (hunk 2 above)
Comment (asmulko) [reply pkg/webui/components/form/field/field.styl]: Looking at #3162 this class was added based on this particular [comment](https://github.com/TheThingsNetwork/lorawan-stack/pull/3162#discussion_r493672041), I only added `line-height` property to it.

**Comment by kschiffer** [comment]
Thanks @asmulko. I went ahead and applied some fixes. The whole label markup is quite messy and I tried to address that a little while fixing the styling issues. This should now also retain the functionality of `cy.findByLabelText()`.

I guess we can merge this once CI is through.

**Review by kschiffer — APPROVED** [review]

**Pull request merged**